test/lit/passes/table64-lowering-features.wast

```diff
@@ -0,0 +1,65 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt %s --enable-memory64 --enable-reference-types --enable-bulk-memory --table64-lowering -S -o - | filecheck %s
+
+(module
+  ;; CHECK:      (type $0 (func))
+
+  ;; CHECK:      (type $1 (func (result i64)))
+
+  ;; CHECK:      (table $t 10 100 funcref)
+  (table $t i64 10 100 funcref)
+
+  (elem (table $t) (i64.const 0) funcref (ref.null func))
```

Comment:
Channelling @tlively , if you give this `elem` a name then the CHECK will appear right on top of it :smile: I always forget that myself.
```suggestion
  (elem $elem (table $t) (i64.const 0) funcref (ref.null func))
```